src/dbnode/generated/proto/annotation/annotation.proto

```diff
@@ -17,3 +23,17 @@ enum MetricType {
     INFO            = 6;
     STATESET        = 7;
 }
+
+enum SourceFormat {
+    // Using 0 for PROMETHEUS_FORMAT because only Prometheus types were being stored before.
+    PROMETHEUS_FORMAT = 0;
```

Comment:
Not sure if we need a separate entry for Open Metrics, or do we map them into Prometheus.

Reply:
I think we could map into prometheus?

Reply:
As discussed, let's call this `OPENMETRICS_FORMAT` given for the purposes of the database the metrics are open metrics? Also we don't have historical metadata going back to differentiate some as open metrics vs Prometheus so probably not a concern trying to capture the differentiation between the two.